packages/cubejs-server/src/index.ts

```diff
@@ -1,9 +1,11 @@
-import { CubejsServer } from './server';
+// import { CubejsServer } from './server';
+import { ServerContainer } from './server/container';
 
 export * from './server';
 
 // Internal staff, don't show it as a public api
 // export * from './command/server';
 // export * from './command/dev-server';
 
-export default CubejsServer;
+export default ServerContainer;
+// export default CubejsServer;
```

Comment:
It's a breaking change because some users continue to use legacy express way
I think We should not do it for now.